### PR TITLE
Ignore invalidError at Init

### PR DESCRIPTION
**Pull request by hajimehoshi**
Updates #324
Updates #292

### CODE REVIEW

**Comment by hajimehoshi** [comment]
Reflected on https://github.com/go-gl/glfw/pull/325#issuecomment-879967448

File: v3.3/glfw/glfw.go [hunk 1]
```diff
@@ -35,9 +35,13 @@ const (
 func Init() error {
 	C.glfwInit()
 	// invalidValue can happen when specific joysticks are used. This issue
-	// will be fixed in GLFW 3.3.5. As a temporary fix, accept this error.
+	// will be fixed in GLFW 3.3.5. As a temporary fix, ignore this error.
 	// See go-gl/glfw#292, go-gl/glfw#324, and glfw/glfw#1763.
-	return acceptError(APIUnavailable, invalidValue)
+	err := acceptError(APIUnavailable, invalidValue)
+	if err.(*Error).Code == invalidValue {
```
Comment (dmitshur) [inline v3.3/glfw/glfw.go]: Is the comma-okay not needed? Won't the type assertion panic when `acceptError` returns a nil error?

Comment (hajimehoshi) [reply v3.3/glfw/glfw.go]: Already updated this PR to use comman-okay

**Review by dmitshur — APPROVED** [review]
Thanks.

**Pull request merged**